compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/JvmBackendFacade.kt

```diff
@@ -53,6 +65,7 @@ object JvmBackendFacade {
         val stubGenerator = DeclarationStubGenerator(
             psi2irContext.moduleDescriptor, psi2irContext.symbolTable, psi2irContext.irBuiltIns.languageVersionSettings, extensions
         )
+        stubGenerator.setIrProviders(irProviders)
         val deserializer = JvmIrLinker(
```

Comment:
Do we need to recreate the `IrProvider` setup here? Why not reuse the one you built earlier?